### ISSUE TITLE
[a11y-defect-1]: VoiceOver iOS users cannot confirm their address

### ISSUE THREAD

**Issue by joshkimux** [department-of-veterans-affairs/va.gov-team#61238]
### Point of contact

joshkimux

### Severity level

1, Launchblocker. Critical. Must be fixed before launch.

### Details

While testing NoD, I noticed an error on VoiceOver iOS when attempting to confirm an address.

https://github.com/department-of-veterans-affairs/va.gov-team/assets/14154792/95fe462c-67f7-4e7d-810e-89cbd51f177b

**Several issues exist on this page:**
- Users are unable to swipe to the "use this address" button, and are effectively trapped within an invisible message that announces "saving changes."
- Should a user manage to escape this (for example, by dragging their fingers to the buttons), double tapping on "use this address" presents another issue
  - When clicked, the button enters a transitional loading state that is announced, but then promptly becomes active again-- reading out the label once more.
  - If a user were to wait long enough, the action will follow through, but this isn't clear or predictable as they may believe nothing occurred as the button is active again with no immediate confirmation of a change

### Reproduction steps

1. Log in as user `vetsgov+test236@id.me` and [reach the contact information page on the NoD flow](https://staging.va.gov/decision-reviews/board-appeal/request-board-appeal-form-10182/introduction) using Safari on iOS
2. Turn on VoiceOver
3. Click "edit" under mailing address
4. Click "save" without making any changes to the address
5. Confirm you are unable to swipe to the buttons
6. Drag your finger to the buttons to access them
7. Double tap on `use this address`
8. Confirm that the `use this address` button becomes inactive, and then active again
9. Confirm a several second delay exists before the user is redirected to the following page
...

### Proposed solution or next steps

I'm unsure what is at the source of this issue, but can provide acceptance criteria

### Acceptance criteria
- [ ] VoiceOver iOS users can swipe to the `use this address` button
- [ ] When clicking on the `use this address` button, it should remain in an inactive loading state until the action is completed

### References, articles, or WCAG support

1.
2.
3.
...

### Type of issue

- [X] Screenreader
- [ ] Keyboard
- [X] Focus
- [ ] Headings
- [ ] Color
- [ ] Zoom
- [ ] Semantics
- [ ] Axe-core
- [ ] Something else
